ocf_data_sampler/torch_datasets/Readme.md

```diff
@@ -0,0 +1,49 @@
+# Torch Datasets
+
+The aim of this folder is to create torch datasets which can easily be used in our ML models and ML deployment.
+
+## PVNet UK Regional
+
+This pipline is for creating GSP predictions which we have used in our PVNet model.
+
+### Init
+
+Starting up this up, we get
+- Time and locations Pairs: A list of all valid time and locations for the data
```

Comment:
Should we call them "t0 times"? Could explain what t0 is as well for completeness